trust-store-trust-policy-specification.md

```diff
@@ -146,37 +145,70 @@ Trust policy for the scenario where ACME Rockets uses artifacts signed by themse
   - **`registryScopes`**(*array of strings*): This REQUIRED property determines which trust policy is applicable for the given artifact.
     The scope field supports filtering based on fully qualified repository URI `${registry-name}/${namespace}/${repository-name}`.
     For more information, see [registry scopes constraints](#registry-scopes-constraints) section.
-  - **`signatureVerification`**(*string*): This REQUIRED property dictates how signature verification is performed. Supported values are `strict`, `permissive`, `audit` and `skip`. Detailed explaination of each value is present [here](#signatureverification-details). A custom level can be defined by referencing a supported level value, and overriding individual validation checks. NOTE: Custom signature verification level will not be supported in `notation` RC1.
-  - **`trustStore`**(*string*): This REQUIRED property specifies a named trust store. Uses the format `{trust-store-type}:{named-store}`. Currently supported values for `trust-store-type` are `ca` and `tsa`. **NOTE**: When support for publicly trusted TSA is available, `tsa:publicly-trusted-tsa` is the default value, and implied without explictly specifying it. If a custom TSA is used the format `ca:acme-rockets,tsa:acme-tsa` is supported to specify it.
+  - **`verificationLevel`**(*string*): This OPTIONAL property dictates how signature verification is performed. Either `verificationLevel` or `verification` property MUST be specified. Supported values are `strict`, `permissive`, `audit` and `skip`. Detailed explaination of each value is present [here](#signatureverification-details).
+  - **`verification`**(*object*): This OPTIONAL property dictates how signature verification is performed using a [custom verification level](#custom-verification-level). Either `verificationLevel` or `verification` property MUST be specified.
+  - **`trustStores`**(*array of string*): This REQUIRED property specifies a set of one or more named trust stores, each of which contain the trusted roots against which signatures are verified. Each named trust store uses the format `{trust-store-type}:{named-store}`. Currently supported values for `trust-store-type` are `ca` and `tsa`.
+    - **NOTE**: When support for publicly trusted TSA is available, `tsa:publicly-trusted-tsa` is the default value, and implied without explictly specifying it. If a custom TSA is used the format `ca:acme-rockets,tsa:acme-tsa` is supported to specify it.
```

Comment:
Should ``ca:acme-rockets,tsa:acme-tsa`` be `["ca:acme-rockets","tsa:acme-tsa"]`?

Reply:
The `ca:acme-rockets,tsa:acme-tsa` format allow you to group related trust stores.

e.g
[
 "ca:acme-rockets,tsa:acme-tsa",
 "ca:wabbit-networks" // Publicly trusted TSA implied
]

Reply:
Should we mention that white spaces before and after of `,` and `"` and  are ignored?